DEVELOPMENT.md

```diff
@@ -25,14 +25,60 @@ $ make init
 
 You may want to run this commands from time to time.
 
-Now you can initialize the development databases with some defaults.
+## Running a development build of The Things Stack
+
+This section explains how to get a bare-bones version of The Things Stack running on your local machine. This will build whatever code is present in your local repository (along with local changes) and run in it using the default ports.
+
+If you want to just run a docker image of The Things Stack, then check the [Installation](https://thethingsstack.io/latest/getting-started/installation/) section of the documentation.
+
+> Note: Make sure that you've run `$ make init` before following this section.
+
+### Pre-requisites
+This section requires [`docker`](https://www.docker.com/get-started) to be installed.
+
+### Steps
+
+1. Build the frontend assets
+
+```bash
+$ ./mage js:build
+```
+
+This will build the frontend assets and place it in the `public` folder.
+
+2. Start the databases
+
+```bash
+$ ./mage dev:dbStart # This requires Docker to be running.
+```
+
+This will start one instance each of `CockroachDB` and `Redis` as Docker containers. To verify this, you can run
+
+```bash
+$ docker ps
+```
+
+3. Initialize the database with defaults.
 
 ```bash
-$ ./mage dev:dbStart   # This requires Docker to be running.
 $ ./mage dev:initStack
 ```
 
-This starts a CockroachDB and Redis database in Docker containers, creates a database, migrates tables and creates a user `admin` with password `admin`.
+This creates a database, migrates tables and creates a user `admin` with password `admin`.
+
+4. Start an development instance of The Things Stack
+
+```bash
+$ go run ./cmd/ttn-lw-stack -c ./config/stack/ttn-lw-stack.yml start
+```
+
+5. Login to the stack via the Console
+
+In a web browser, navigate to `http://localhost:1885/` and login using credentials from step 3.
+
+6. Customizing configuration
+
+To customize the configuration, edit the `/config/stack/ttn-lw-stack.yml` file. The configuration is documented in the [Configuration Reference](https://thethingsstack.io/latest/reference/configuration/).
```

Comment:
Maybe better to recommend creating a copy and pointing to that?

Reply:
Indeed. Actually my original idea was that the script would make a copy of the config in `.env` and that's freely editable.